README.md

```diff
@@ -12,7 +12,7 @@ To get a quick overview of the config file, see an example file [here](https://g
 
 ## Model Specification
 
-A model entry in the bioimage.io model zoo is defined by a configuration file `<model name>.model.yaml`.
```

Comment:
isn't the `<model name>` optional, as in it might be there?

Reply:
Not exactly, we said that if the yaml file is inside a model package, then we should remove the model name part, but if it's outside, e,g. on github, we should have it.

Reply:
Yes, I thought it was better to remove `<model name>` for now, because we haven't defined the differences between the config on github vs in the zip bundle yet, and we also haven't defined the zip bundle (see also #58).